extensions/amp-story/1.0/amp-story-page-attachment.css

```diff
@@ -139,6 +139,7 @@ amp-story-page[active] .i-amphtml-story-page-open-attachment {
   pointer-events: none !important;
   z-index: 3 !important;
   animation: open-attachment-fly-in 0.3s cubic-bezier(0.4, 0.0, 0.2, 1) both !important;
+  -webkit-touch-callout: default !important;
```

Comment:
So strange that safari needs this.
I suppose the `-webkit-` prefix hints at this but I wonder if a comment would be helpful.

Reply:
So I somewhat explained this in the description of this PR, but `amp-story` specifically disables this on the story itself by setting it to none, and this cascades down to all the elements of a story (so you can't long-press anywhere). Then, if we want to enable it specifically for this element, we need to add this here. It was a bit tricky so I'll add a comment!

Reply:
Since the next person to see this code will likely not see this PR it's great to have the comment!